### PR TITLE
Added a model for ReviewConfiguration

### PR DESCRIPTION
**Pull request by lzbotha**
Added a ReviewConfiguration model as per:
https://github.com/deep-learning-indaba/Baobab/issues/613

### CODE REVIEW

**Comment by amritpurshotam** [comment]
Looking now

File: api/app/reviews/models.py [hunk 1]
```diff
@@ -100,4 +100,16 @@ def __init__(self,
...
-        self.value = value
\ No newline at end of file
+        self.value = value
+
+
+class ReviewConfiguration(db.Model):
+    id = db.Column(db.Integer(), primary_key=True)
+    review_form_id = db.Column(db.Integer(), db.ForeignKey('review_form.id'), nullable=False)
+    num_reviews_required = db.Column(db.Integer())
+    num_optional_reviews = db.Column(db.Integer())
```
Comment (amritpurshotam) [inline api/app/reviews/models.py]: I think specify `nullable=False` here as well and then do another migration (having 2 will be fine)

File: api/app/reviews/models.py [hunk 2]
```diff
@@ -100,4 +100,16 @@ def __init__(self,
...
+    drop_optional_question_id = db.Column(db.Integer(), db.ForeignKey('review_question.id'), nullable=False)
+    drop_optional_agreement_values = db.Column(db.String(), nullable=False)
```
Comment (avishkar58) [inline api/app/reviews/models.py]: These two fields should be nullable

Comment (amritpurshotam) [reply api/app/reviews/models.py]: Oh you got to it first 😅

File: api/app/reviews/models.py (hunk 1 above)
Comment (lzbotha) [reply api/app/reviews/models.py]: I think the default is `nullable=False`. Happy to make this explicit though.

Comment (lzbotha) [reply api/app/reviews/models.py]: Particularly since I did it on all the other fields 😓

Comment (amritpurshotam) [reply api/app/reviews/models.py]: Na if you look at the migration, they were set to be nullable.
Unless do you know of a way to configure SQL Alchemy to make fields not nullable by default?

Comment (lzbotha) [reply api/app/reviews/models.py]: You're completely right.

**Comment by amritpurshotam** [comment]
@lzbotha Cool, first closed PR for the day!

**Pull request merged**